build/wikia/transformers/transformPolarity.mjs

```diff
@@ -1,6 +1,4 @@
-'use strict';
-
-const POLARITIES = require('./polarities');
+import { POLARITIES } from './polarities.mjs';
```

Comment:
```suggestion
import POLARITIES from './polarities.mjs';
```

Reply:
use default where possible, not the destructured ones

Reply:
why?

Reply:
as stated before with destructuring it's more clear that i'm not renaming anything and it's what is expected to be. Different from default export which could two different names in two modules. I think this is bad style and design.

Reply:
yes, but our styling that i've had in place for a while specifies that we should prefer defaults for exports, so why should we do the opposite for imports?

Reply:
> yes, but our styling that i've had in place for a while specifies that we should prefer defaults for exports, so why should we do the opposite for imports?

Maybe i'll open a PR on the styling then.